tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -376,7 +377,7 @@ def test_cpu_wrmsr_snapshot(
     that MSRs retain their values after restoring from a snapshot.
 
     This function makes MSR value modifications according to the
-    ../resources/tests/msr/wrmsr_list.txt file.
+    ../msr/wrmsr_list.txt file.
```

Comment:
not `../msr/wrmsr_list.txt` but `./data/msr/wrmsr_list.txt`?

Reply:
Good catch!